pkg/plugins/backendplugin/grpcplugin/grpc_plugin.go

```diff
@@ -58,16 +58,9 @@ func (p *grpcPlugin) Start(ctx context.Context) error {
 		return err
 	}
 
-	if p.client.NegotiatedVersion() > 1 {
-		p.pluginClient, err = newClientV2(p.descriptor, p.logger, rpcClient)
-		if err != nil {
-			return err
-		}
-	} else {
-		p.pluginClient, err = newClientV1(p.descriptor, p.logger, rpcClient)
-		if err != nil {
-			return err
-		}
+	p.pluginClient, err = newClientV2(p.descriptor, p.logger, rpcClient)
```

Comment:
I think an error returned if `p.client.NegotiatedVersion() < 2` would be nice. Something along the lines "Plugin uses a version of Grafana's backend plugin system that is no longer supported"